src/env.cc

```diff
@@ -1096,6 +1096,12 @@ void Environment::InitializeCompileCache() {
       dir_from_env.empty()) {
     return;
   }
+  if (!options()->experimental_policy.empty()) {
+    Debug(this,
```

Comment:
Could this be a warning instead?

Reply:
The environment variable is designed to be as silent as possible. I don't think there's anything to warn - you don't get a speed up and that is all.